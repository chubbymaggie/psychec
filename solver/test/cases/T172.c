
void f()
{
    struct S const* data;

    data = ent->weit;
    data->mode;

    g(ent->weit);
    g(data);

}
