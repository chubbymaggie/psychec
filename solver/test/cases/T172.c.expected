/* Forward declarations */
typedef  struct TYPE_2__   TYPE_1__ ;

/* Type definitions */
typedef  struct S {int /*<<< orphan*/  mode; } const S ;
struct TYPE_2__ {struct S const* weit; } ;

/* Variables and functions */
 TYPE_1__* ent ; 
 int /*<<< orphan*/  g (struct S const*) ; 

